### PR TITLE
Add <mark> example

### PR DESCRIPTION
**Pull request by Elchi3**
Example for https://developer.mozilla.org/en-US/docs/Web/HTML/Element/mark

### CODE REVIEW

File: live-examples/html-examples/inline-text-semantics/mark.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>Search results for <strong>"tutorials"</strong>:</p>
+
+<ul>
+  <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial">Canvas <mark>tutorials</mark></a>. How to use the canvas element to draw 2D graphics, starting with the basics.</li>
+
```
Comment (schalkneethling) [inline live-examples/html-examples/inline-text-semantics/mark.html]: Nit: No empty line between list items.

**Comment by Elchi3** [comment]
Updated. Also changed the title to no longer use the encoded stuff.

**Comment by Elchi3** [comment]
Wait, if I'm not using any CSS, I shouldn't use the tabbed template but the html template, right?

**Comment by Elchi3** [comment]
Ok!
Given this is about `<mark>`, I think it would be good to just have an empty rule so that people can get right into playing with the CSS for `<mark>` if they want to.

**Pull request merged**